SWLOR.Game.Server/Feature/StatusEffectDefinition/CreepingTerrorStatusEffectDefinition.cs

```diff
@@ -36,15 +37,20 @@ private void CreepingTerror()
                 })
                 .TickAction((source, target, effectData) =>
                 {
+                    var willBonus = GetAbilityModifier(AbilityType.Willpower, source);
                     var level = (int)effectData;
-                    var dmg = level * 4; // (4/8/12)
+                    var dmg = level * 4 + (willBonus * 1); // (4/8/12)
```

Comment:
Multiplication by 1 is unnecessary here. Also the comment is incorrect now with your changes. Either remove the comment or update it to be accurate.